make-rules/shared-macros.mk

```diff
@@ -26,8 +26,8 @@
 # whenever PATH is to be defined there:
 #     PATH = $(PATH.illumos)
 #     PATH = $(PATH.gnu)
-PATH.illumos=$(USRBINDIR):$(GNUBIN):$(USRSBINDIR):$(PERL5BINDIR)
-PATH.gnu=$(GNUBIN):$(USRBINDIR):$(USRSBINDIR):$(PERL5BINDIR)
+PATH.illumos=$(USRBINDIR$(BITS)):$(USRBINDIR):$(GNUBIN):$(USRSBINDIR$(BITS)):$(USRSBINDIR):$(PERL5BINDIR)
+PATH.gnu=$(USRBINDIR$(BITS)):$(GNUBIN):$(USRBINDIR):$(USRSBINDIR$(BITS)):$(USRSBINDIR):$(PERL5BINDIR)
```

Comment:
Shouldn't $(GNUBIX) first (before $(USRBINDIR$(BITS)))? Otherwise GNU wouldn't be used...

Reply:
Yes. Also we have unused 64-bit binaries in /usr/gnu/bin and that's annoying. But let us leave it for later.

Reply:
Fixed.